build.gradle

```diff
@@ -112,19 +112,41 @@ task buildAwsLc {
     }
 
     doLast {
-        exec {
-            workingDir awslcSrcPath
-            executable cmakeBin
-            args "-B${cMakeBuildDir}"
-            args '-DBUILD_SHARED_LIBS=ON'
-            args '-DCMAKE_BUILD_TYPE=RelWithDebInfo'
-            args "-DCMAKE_INSTALL_PREFIX=${sharedObjectOutDir}"
-            args "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
+        // TODO: Remove this when we finally deprecate gcc4.1.2/Cmake 3.9 internally.
+        // Behavior of how older versions of Cmake take in the the build and source
+        // directory arguments is slightly different.
```

Comment:
nit: might want to make a note that we don't care about FIPS in legacy build because legacy build is **only** for checking compiler compatibility.